### PR TITLE
Remove deleting files at startup

### PR DESCRIPTION
**Pull request by mmnosek**
Description in #56

### CODE REVIEW

**Comment by warpech** [comment]
I am reviewing. Is it ok if I merge it to `develop`? Then it will not be available in SWO until @aloscha merges his changes too.

**Comment by warpech** [comment]
I am reviewing. Is it ok if I merge it to `develop`? Then it will not be available in SWO until @aloscha merges his changes too (should be on Monday, he says).

**Comment by Mnosek** [comment]
I will also cherry-pick it to our SWO branch as it is completely separated.

**Comment by warpech** [comment]
Works nice but looks bad in Starcounter stable (because of a bug in Starcounter stable with unnamed slots). Could you please add slot names?

2016-11-17 12_35_41-launcher

File: src/Images/wwwroot/Images/viewmodels/SettingsPage.html [hunk 1]
```diff
@@ -6,5 +6,6 @@
         <label class="control-label" style="font-size: 12pt; margin: 10px 0 0 0;">Maximum file size (bytes)</label>
         <input type="number" value="{{model.MaximumFileSize$::input}}" class="form-control" style="width: 50%"/>
         <button class="btn btn-small btn-success" style="margin-top: 20px;" onclick="++this.value;" value="{{model.Save$::click}}">Save</button>
+        <button class="btn btn-small btn-danger" style="margin-top: 20px;" onclick="++this.value;" value="{{model.CleanUpFiles$::click}}">Clean up files</button>
```
Comment (Mnosek) [inline src/Images/wwwroot/Images/viewmodels/SettingsPage.html]: It's bootstrap, we don't want to change them.

Comment (aloscha) [inline src/Images/wwwroot/Images/viewmodels/SettingsPage.html]: yeah, but I mean this one `style="margin-top: 20px;"`

**Comment by Mnosek** [comment]
Fixed, I've added starcounter-composition to settings page.

**Comment by Mnosek** [comment]
ping @warpech

**Comment by warpech** [comment]
Looks good! Merging

**Pull request merged**